src/Android/Avalonia.Android/Platform/Storage/AndroidStorageItem.cs

```diff
@@ -392,6 +392,8 @@ public AndroidStorageFile(Activity activity, AndroidUri uri, AndroidStorageFolde
     public Task<Stream> OpenReadAsync() => Task.FromResult(OpenContentStream(Activity, Uri, false)
         ?? throw new InvalidOperationException("Failed to open content stream"));
 
+    public Stream OpenRead() => OpenReadAsync().Result;
```

Comment:
Do the reverse, put the logic in `OpenRead()` and have `OpenReadAsync()` wrap that with `Task.FromResult()`. Right now this is essentially `Task.FromResult(...).Result`, we don't need that task.